src/libfsm/endids.h

```diff
@@ -41,6 +67,16 @@ void
 fsm_endid_iter(const struct fsm *fsm,
```

Comment:
Should we remove the non-bulk iterator later? It's an internal interface, during integration with the new capture code (which also updates the "consolidate/copy/merge/etc. end metadata" operation during transformations) I'll see if anything actually needs the non-bulk version.

Reply:
They're internal, so probably.  If the implementation changes to something that makes the bulk versions undesirable, someone will have to revisit everything anyway, so keeping the non-bulk iterators around probably doesn't help much.